tests/test_rabbit_watcher.py

```diff
@@ -0,0 +1,44 @@
+import os
+import time
+from unittest import TestCase
+
+import casbin
+from rabbitmq_watcher import new_watcher
+
+
+def get_examples(path):
+    examples_path = os.path.split(os.path.realpath(__file__))[0] + "/../examples/"
+    return os.path.abspath(examples_path + path)
+
+
+class TestConfig(TestCase):
+    def test_update_rabbitmq_watcher(self):
+        watcher = new_watcher()
+        watcher.channel.basic_publish(
+            exchange="", routing_key="casbin-policy-updated", body=str(time.time())
+        )
+        assert watcher.update() is True
+
+    def test_with_enforcer(self):
+        def _test_update_callback(msg):
+            print("update callback, msg: {}".format(msg))
```

Comment:
Please add an assertion in the callback, please ensure that the message is received in the callback().